### PR TITLE
Updates for version 0.6.0

### PR DESCRIPTION
**Pull request by SueChaplain**
- terminology changes, following removal of collections
- brand changes
- conversion to markdown format

Signed-off-by: Sue Chaplain <sue_chaplain@uk.ibm.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+permalink: /guides/use-appsody-cli/
+layout: guide-markdown
+title: Developing microservice applications with the CLI
+duration: 20 minutes
+releasedate: 2019-12-03
+description: Learn about the common CLI commands that you'll use to develop applications
+tags: ['CLI']
+guide-category: basic
+---
+
```
Comment (davehobbs) [inline README.md:11]:
```suggestion
<!-- Note:
> This repository contains the guide documentation source. To view
> the guide in published form, view it on the [website](https://kabanero.io/guides/{projectid}.html).
-->

```

**Review by davehobbs — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
+---
+permalink: /guides/use-appsody-cli/
+layout: guide-markdown
+title: Developing microservice applications with the CLI
+duration: 20 minutes
+releasedate: 2019-12-03
```
Comment (stephenkinder) [inline README.md:6]: not sure the purpose of this field, is there an updatedate: also?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+
+### Adding a customized application stack repository
+
+**CLI command:** `appsody repo add <repo-name> <URL>`
+
+As a developer, your enterprise architect might provide you with a URL that points to a set of customized application stacks for your organization. This configuration is defined by an `index.yaml` file.
+
+To add this configuration information to your local development environment, use the `appsody repo add <repo-name> <URL>` command, supplying a name for the repository and the URL that contains the `index.yaml` file.
+
+In this example, you will add a new repository `abc-stacks` that has a stack configuration URL https://github.com/abc.inc/stacks/index.yaml.
```
Comment (stephenkinder) [inline README.md]: abc is actually a well-known company in the US?  I think typically we use `acme` ?

File: README.md [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+
+In earlier sections of this guide you created a `nodejs-express` project with the default (`simple`) template, which provides a sample test as a starting point. Take a look at the sample test in the `my-project/test/test.js` file. You can update this file to suit your test requirements.
+
+Now try running the `appsody test` command for your project. The results from the test suite are included in the output.
+
+The testing uses constructs that are familiar to the programming language or framework on which the stack is based. You can add your own tests or switch to your preferred testing framework. Node.js application stacks use the [Mocha](https://mochajs.org/) test framework as default. If you want to use a different test framework, update the `npm test` command in your project `package.json` file.
```
Comment (stephenkinder) [inline README.md:205]: where do you do the `npm test` upgrade?  the application stack container, or is this the local environment?

**Review by stephenkinder — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (SueChaplain) [reply README.md:6]: @alohr51 - I'm not aware of any YAML values for "date updated" - can you confirm please?

File: README.md (hunk 3 above)
Comment (SueChaplain) [reply README.md]: I've switched the references to "acme" - I don't think there is a precedent, but that's as good as any.

File: README.md (hunk 4 above)
Comment (SueChaplain) [reply README.md:205]: If you update the `package.json` file the changes apply both to a local development container and the deployment container.  Do you think this needs to be called out specifically @stephenkinder ?

Comment (stephenkinder) [reply README.md:205]: @SueChaplain I'll leave it to you as to whether it belongs here, elsewhere or nowhere.

**Pull request merged**